README.md

```diff
@@ -161,6 +161,18 @@ The `UMF_BUILD_LIBUMF_POOL_SCALABLE` option has to be turned `ON` to build this
 2) Required packages:
    - libtbb-dev (libraries: libtbbmalloc.so.2)
 
+### Memspaces (Linux-only)
+
+TODO: Add general information about memspaces.
+
+#### Default memspace
+
+Memspace backed by NUMA node discovery. Can be retrieved using umfMemspaceDefaultGet.
```

Comment:
Can we describe more precisely what “dafault” means (e.g. all available NUMA nodes)? Or we intentionaly do not specify this for some abstraction purposes.

Reply:
I've changed the brief about `default` memspace. The info that it's backed by all available NUMA nodes is now transparent.